.github/scripts/.helm-tests/local-global-precedence/values.yaml

```diff
@@ -0,0 +1,30 @@
+lifecycleOperator:
+  lifecycleOperator:
+    image:
+      registry: "local-registry.io" # will be used
+      imagePullPolicy: Never # will be used
+  scheduler:
+    image:
+      registry: "local-registry.io" # will be used
+      imagePullPolicy: Never # will be used
+metricsOperator:
+  image:
+    registry: "local-registry.io" # will be used
+    imagePullPolicy: Never # will be used
+certManager:
+  image:
+    registry: "local-registry.io" # will be used
+global:
+  imageRegistry: "global-registry.io" # will not be used
+  imagePullSecrets:
+    - name: registry-secret
+    - second-secret
+  imagePullPolicy: Always # will not be used
+  commonLabels:
+    globalLabel1: test1 # will be used
```

Comment:
if we introduce label with the same key on global and also component level, let's say metrics-operator, which value will be used? I would say let's add a test case for that

Reply:
there's is actually no way to define levels on a local level 😅

Reply:
let's take a use case:

```
global:
    label1: test1
    
metricsOperator:
    operatorService:
        annotations: label1: test2
```

which value will be used? test1 or test2?

Reply:
or is it not valid to compare these two?

Reply:
aha! i only checked for labels and there no way for those to be set, but annotations are possible. i'll add a test for that